### ISSUE TITLE
Typo in auto icons option object type definition

### ISSUE THREAD

**Issue by nralbrecht** [wxt-dev/wxt#881]
### Describe the bug

The `AutoIconsOptions` interface contains the [`baseIconPath`](https://github.com/wxt-dev/wxt/blob/aea123890f688c658ed16c2d2fd711146fa34e26/packages/auto-icons/src/index.ts#L85C3-L85C15) field but in the code `baseIconsPath` is used instead ([here](https://github.com/wxt-dev/wxt/blob/aea123890f688c658ed16c2d2fd711146fa34e26/packages/auto-icons/src/index.ts#L14C7-L14C20), [here](https://github.com/wxt-dev/wxt/blob/aea123890f688c658ed16c2d2fd711146fa34e26/packages/auto-icons/src/index.ts#L22) and [here](https://github.com/wxt-dev/wxt/blob/aea123890f688c658ed16c2d2fd711146fa34e26/packages/auto-icons/src/index.ts#L23)).

Depending which name I use this leads to either the default path being used or a type error in `wxt.config.ts`.

### Reproduction

Work in progress repository at the commit that ads `@wxt-dev/auto-icons`:

https://github.com/nralbrecht/youtube-hide-controls/tree/d4838a40e734ee03e808bc0dadcd26358e50f235

### Steps to reproduce

1. `bun install`
2. Generate types with `bun run build`
3. Notice type error in `wxt.config.ts`

### System Info

```shell
System:
    OS: Linux 6.9 Arch Linux
    CPU: (16) x64 AMD Ryzen 7 3700X 8-Core Processor
    Memory: 9.42 GB / 31.29 GB
    Container: Yes
    Shell: 5.9 - /bin/zsh
  Binaries:
    Node: 18.20.3 - /bin/node
    Yarn: 1.22.22 - /bin/yarn
    npm: 10.8.1 - /bin/npm
    bun: 1.1.20 - /bin/bun
  npmPackages:
    wxt: ^0.18.15 => 0.18.15 
    @wxt-dev/auto-icons: ^1.0.0 => 1.0.0
```

### Used Package Manager

bun

### Validations

- [X] Read the [Contributing Guidelines](https://github.com/wxt-dev/wxt/blob/main/CONTRIBUTING.md).
- [X] Read the [docs](https://wxt.dev/guide/installation.html).
- [X] Check that there isn't [already an issue](https://github.com/wxt-dev/wxt/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/wxt-dev/wxt/discussions) or join our [Discord Chat Server](https://discord.gg/ZFsZqGery9).
- [x] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by Timeraa** [comment]
Hey, I fixed that issue and from what I see it's already released with 1.0.1

**Comment by nralbrecht** [comment]
Great it works now. Thank you :+1:
